pkg/api/v1/api.proto

```diff
@@ -58,6 +59,28 @@ message GetCommitInfoResponse {
   bool load_more = 7;
 }
 
+enum GitSyncStatus {
+  SYNC_UNKNOWN = 0;
+  UNSYNCED    = 1;
+  SYNCED      = 2;
+  SYNC_FAILED = 3;
+}
+
+message GetGitSyncStatusRequest {
+
+}
+
+message GetGitSyncStatusResponse {
+  repeated EnvApp unsynced = 1;
+  repeated EnvApp sync_failed = 2;
+}
+
+message EnvApp {
+  string application_name = 1;
+  string environment_name = 2;
+  GitSyncStatus status    = 3;
```

Comment:
I don't understand. If these are going to be in the `unsynced` or `sync_failed` list, then why would we need the status field here? We already know that they're unsynced or sync_failed!